### DISCUSSION TITLE
Make unfurling optional

### DISCUSSION THREAD

**Discussion by OndrejSpanel** [community/community#43484] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

Currently links are unfurled in [task-lists](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists), but kept short otherwise.

Some people would like them unfurled everywhere (https://github.com/community/community/discussions/4260), sometimes you do not want this functionality even in task-lists (https://github.com/community/community/discussions/4321)

A dedicated syntax to be able to request unfurling was suggested, like #4260+. There could also be another syntax to disable unfurling in task lists be appending a minus sing, like #4321-.

If trailing plus and minus are not possible for some technical reasons, other syntax can be used. but having the functionality accessible in some way would be really helpful.

**Comment by zyv** [comment]
Cross-reference:

* #41837
